### PR TITLE
moving 2nd line of beta warning to h3

### PR DESCRIPTION
**Pull request by leahbannon**
@noahmanger what do you think?

### CODE REVIEW

File: templates/layouts/main.html [hunk 1]
```diff
@@ -61,8 +61,7 @@
   <header class="site-header">
     <div id="notice" aria-hidden="true" class="section--blue">
       <a id="notice-close" class="js-action js-hide link--icon" tabindex="0" data-hides="notice"><i class="ti-close"></i></a>
-      <h3>This is a work in progress</h3>
-      <p>This site is in <a href="https://18f.gsa.gov/dashboard/stages/#beta" target="_blank">beta</a>. <a href="/#progress">Find out what’s here and what’s missing</a>.</p>
+      <h3>This is a work in progress... this site is in <a href="https://18f.gsa.gov/dashboard/stages/#beta" target="_blank">beta</a>. <a href="/#progress">Find out what’s here and what’s missing</a>.</h3>
```
Comment (noahmanger) [inline templates/layouts/main.html]: Can we ditch the ellipsis? Seems too informal. 

Also, h3 feels real big. How about h4?

**Comment by leahbannon** [comment]
> Can we ditch the ellipsis? Seems too informal.

I'm not married to ellipsis, but I think it needs some punctuation and "This is a work in progress. This site is in beta." seemed a bit stilted. But maybe it's not that bad?

> Also, h3 feels real big. How about h4?

I don't have it set up locally, so I can't preview, but the current version with "This is a work in progress." alone looked... big and lonely?

**Comment by emileighoutlaw** [comment]
@leahbannon @noahmanger Thoughts on combining the two into "This beta site is a work in progress" ??

**Comment by leahbannon** [comment]
How about something that looks like...

### Welcome to OpenFEC. This [beta]() site is work in progress.
#### [Find out what's here and what's missing.]()
Contact us blah blah blah.

**Pull request merged**